tdct/deconvolution.py

```diff
@@ -0,0 +1,864 @@
+import sys
+import os
+
+import tifffile
+if getattr(sys, 'frozen', False):
+    # programm runs in a bundle (pyinstaller)
+    execdir = sys._MEIPASS
+else:
+    execdir = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(execdir)
+
+import numpy as np
+from PyQt5 import QtWidgets
+from . import clrmsg
+from . import TDCT_debug
+
+try:
+    import tifffile as tf
+except:
+    sys.exit("Please install tifffile, e.g.: pip install tifffile")
+
+debug = TDCT_debug.debug
+
+import scipy.signal
+import scipy.fft
+
+#Internal helper class to handle progress bar
+class _progrBarHandle():
```

Comment:
According to python PEP8 class names should be capitalised like ProgrBarHandle. Maybe rename this class to fit python convention. But if you prefer it to be hidden you could still use _ProgrBarHandle.

https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles